src/rules/requireJsdoc.js

```diff
@@ -119,20 +119,22 @@ const getOptions = (context) => {
         return false;
       }
 
-      return Object.keys(baseObj.properties).reduce((obj, prop) => {
+      const properties = {};
+      Object.keys(baseObj.properties).forEach((prop) => {
```

Comment:
Any reason for using `forEach` over `for ... of`?

Reply:
I am just in the habit of using `forEach`, as I like that I can just move the function out of the `forEach` if needed, changing only the reference within `forEach`. It feels more easily composable than `for...of`.